spyder_kernels/customize/spyderpdb.py

```diff
@@ -123,12 +121,13 @@ def default(self, line):
             self.print_exclamation_warning()
             self.error("Unknown command '" + line.split()[0] + "'")
             return
-        # Disallow the use of %debug magic in the debugger
```

Comment:
telling the user to replace %debug by debug is a waste of his time, just do it ourself